src/plugins/Dashboard/FileItem.js

```diff
@@ -52,16 +54,23 @@ module.exports = function fileItem (props) {
                         }"
                   onclick=${(ev) => {
                     if (isUploaded) return
+                    if (error) {
+                      props.retryUpload(file.id)
+                      return
+                    }
                     if (props.resumableUploads) {
                       props.pauseUpload(file.id)
                     } else {
                       props.cancelUpload(file.id)
                     }
                   }}>
```

Comment:
Might be good to move these largeish event handlers into a `function onclick (ev) {}` and do `onclick=${onclick}` at some point, a la

```js
function view () {
  function onclick () {
    alert('boo! 👻')
  }
  return html`<button onclick=${onclick}>click</button>`
}
```

Reply:
👍